src/vmm/src/logger/logging.rs

```diff
@@ -301,13 +349,21 @@ mod tests {
         assert_eq!(LevelFilter::from_str("Warn"), Ok(LevelFilter::Warn));
         assert_eq!(LevelFilter::from_str("Error"), Ok(LevelFilter::Error));
 
+        assert_eq!(LevelFilter::from_str("off"), Ok(LevelFilter::Off));
+        assert_eq!(LevelFilter::from_str("trace"), Ok(LevelFilter::Trace));
+        assert_eq!(LevelFilter::from_str("debug"), Ok(LevelFilter::Debug));
+        assert_eq!(LevelFilter::from_str("info"), Ok(LevelFilter::Info));
+        assert_eq!(LevelFilter::from_str("warn"), Ok(LevelFilter::Warn));
+        assert_eq!(LevelFilter::from_str("error"), Ok(LevelFilter::Error));
+
```

Comment:
I think we can merge this entire test into the one above, by having the `["off",...]` array contain tuples of `(str, LevelFilter)` where the second argument contains the expected outcome of parsing